private-templates-service/adaptivecards-templating-service/src/models/models.ts

```diff
@@ -1,27 +1,27 @@
 export interface IUser {
-    id?: string;
-    team: string[];
-    org: string[];
-    email: string;
-  }
-  
+	id?: string;
```

Comment:
Set up vscode to use spaces instead of tabs - can ask @guptaharsh25 about how to set it up